flink-libraries/flink-table/src/main/scala/org/apache/flink/table/runtime/aggregate/GeneratedAggregations.scala

```diff
@@ -100,6 +108,17 @@ abstract class GeneratedAggregations extends Function {
     *                     aggregated results
     */
   def resetAccumulator(accumulators: Row)
+
+  /**
+    * Cleanup for the accumulators.
+    */
+  def cleanup()
+
+  /**
+    * Tear-down method for [[org.apache.flink.table.functions.AggregateFunction]].
+    * It can be used for clean up work. By default, this method does nothing.
+    */
+  def close()
```

Comment:
The close() method is corresponding to open method. And it will be called when open was called before, e.g. ProcTimeUnboundedOver's close() method. Otherwise, the udagg's close will not be called. 

I modify the case testGroupAggregateWithStateBackend to test the close was called.

what do you think?

Reply:
Sorry, I overlooked the `close()` calls. If the method is used, we should keep it of course.